### PR TITLE
feat: custom images guide

### LINKED ISSUE #568: new tutorial: How to create and run a custom product image
This tutorial should: 

- explain where all the images are
- show how to use our images as base images
- give an example of where to deploy this (i.e. custom registry?)
- show how to use this custom image in a Stacklet definition
- show how to update a custom image

```
# Dockerfile
FROM docker.stackable.tech/stackable/spark-k8s:3.5.0-stackable23.11.0
```

### PR DESCRIPTION
**Pull request by fhennig**
closes #568 

this is a _guide_, and I added it to a new section, that I will expand on in the future. some of the tutorials will be moved here, as they are actually guides.

you can read more about the difference here: https://diataxis.fr/tutorials-how-to/

### CODE REVIEW

**Comment by fhennig** [comment]
preview of the page: https://deploy-preview-595--stackable-docs.netlify.app/home/nightly/guides/custom-images

File: modules/guides/pages/custom-images.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+=== Use your customized image in your Stacklet definition
+
+To use the customized image in your Stacklet, adjust the xref:concepts:product-image-selection.adoc[] section in your Stacklet configuration.
+Use the `custom` field to specify your image:
+
+[source,yaml]
+spec:
+  image:
+    custom: my.corp/stackable/hive:3.1.3-stackable24.3.0-mysql
+    productVersion: 3.3.1
```
Comment (razvan) [inline modules/guides/pages/custom-images.adoc:54]:
```suggestion
    productVersion: 3.1.3
```

File: modules/guides/pages/custom-images.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Note that you still need to specify the product version was used in the image, as the operator needs to know which version the product is in to configure it correctly.
```
Comment (razvan) [inline modules/guides/pages/custom-images.adoc:56]:
```suggestion
Note that you still need to specify the product version was used in the image for the operator  to configure it correctly.
```

**Review by razvan — APPROVED** [review]
some suggestions

**Comment by fhennig** [comment]
@razvan thanks!

**Review by razvan — APPROVED** [review]
lgtm

**Pull request merged**